glassfish-runner/jpa-platform-tck/pom.xml

```diff
@@ -332,6 +332,17 @@
             <artifactId>derbytools</artifactId>
             <version>10.15.2.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.glassfish.main.deployment</groupId>
```

Comment:
Is this just temp to help with debugging?

Reply:
Yes, leftover from looking at adding a glassfish annotation handler to the test deployment.